website/content/docs/internals/plugins/plugin-architecture.mdx

```diff
@@ -144,16 +123,33 @@ configured in the plugin catalog. Finally Vault runs the command configured in
 the catalog, sending along the JWT formatted response wrapping token and mlock
 settings. Like Vault, plugins support [the use of mlock when available](/docs/configuration#disable_mlock).
 
-~> Note: If Vault is configured with `mlock` enabled, then the Vault executable and each
-plugin executable in your [plugins directory](/docs/internals/plugins#plugin-directory) must be
-given the ability to use the `mlock` syscall.
+~> Note: If Vault is configured with `mlock` enabled, then the Vault executable
+and each plugin executable in your [plugins directory](/docs/internals/plugins/plugin-architecture#plugin-directory)
+must be given the ability to use the `mlock` syscall.
 
 ### Plugin Upgrades
 
-Plugins may be updated by registering and reloading them. More details on the
-upgrade procedure can be found in [Upgrading Vault Plugins](/docs/upgrading/plugins).
+External plugins may be updated by registering and reloading them. More details
+on the upgrade procedure can be found in
+[Upgrading Vault Plugins](/docs/upgrading/plugins).
+
+## Plugin Multiplexing
+
+As of Vault 1.10, database plugins can be made to implement plugin multiplexing
+which allows a single plugin process to be used for multiple database
+connections. This single process, per database plugin, will be multiplexed
+across all Vault namespaces for mounts of this type. Multiplexing a plugin
+does not affect the current behavior of existing plugins.
+
+To enable multiplexing, the plugin must be compiled with the `ServeMultiplex`
+function call from Vault's `dbplugin` package. At this time, there is no
+opt-out capability for plugins that implement multiplexing. To use a
+non-multiplexed plugin, simply run an older version of the plugin. i.e. the
```

Comment:
Consider removing the adverb 'simply' and just say...run an older version of the plugin.